rainier-core/src/main/scala/com/stripe/rainier/core/WAIC.scala

```diff
@@ -0,0 +1,112 @@
+package com.stripe.rainier.core
+
+import com.stripe.rainier.compute._
+import com.stripe.rainier.ir._
+
+case class WAIC(lppd: Double, pWAIC: Double) {
```

Comment:
want to add a comment to explain this for the uninitiated what `WAIC` stands for maybe with a link to read more?